exporters/inmemory/src/main/java/io/opentelemetry/exporters/inmemory/InMemoryTracing.java

```diff
@@ -43,71 +45,59 @@
  *
  * @since 0.1.0
  */
-public final class InMemoryTracing {
-  private final TracerSdkProvider tracerSdkProvider;
-  private final InMemorySpanExporter exporter;
-
+@AutoValue
```

Comment:
Autovalue seems like a little bit of overkill for this, since we don't really care about equals/hashcode/tostring, but I'm not opposed to it if people like standardizing on it for simple immutable classes.

Reply:
I don't like writing all getters/setters variables, etc. Lazy :)